diff --git a/node_modules/fastfile/src/fastfile.js b/node_modules/fastfile/src/fastfile.js
index 25db11e..8f3ef30 100644
--- a/node_modules/fastfile/src/fastfile.js
+++ b/node_modules/fastfile/src/fastfile.js
@@ -2,7 +2,8 @@
 import { open } from "./osfile.js";
 import * as memFile from "./memfile.js";
 import * as bigMemFile from "./bigmemfile.js";
-import { O_TRUNC, O_CREAT, O_RDWR, O_EXCL, O_RDONLY } from "constants";
+
+const { O_TRUNC, O_CREAT, O_RDWR, O_EXCL, O_RDONLY } = "constants";
 
 const DEFAULT_CACHE_SIZE = (1 << 16);
 const DEFAULT_PAGE_SIZE = (1 << 13);
